charts/govuk-jobs/templates/govuk-mirror-sync-cronjob.yaml

```
@@ -114,5 +114,5 @@ spec:
                   /s5cmd sync --size-only "/data/${ASSETS_DOMAIN}" "s3://${BUCKET_NAME}/"
 
                   # Upload file for checking mirror freshness
-                  touch "/data/${WWW_DOMAIN}/last-updated.txt"
+                  date "+%Y-%m-%d %H:%M:%S" > "/data/${WWW_DOMAIN}/last-updated.txt"
```

optional: might as well use a standard format and include the tz
```suggestion
                  date --iso-8601=seconds > "/data/${WWW_DOMAIN}/last-updated.txt"
```

---

ah I was too slow haha

---

https://github.com/alphagov/govuk-helm-charts/pull/2124